docs/check.py

```diff
@@ -19,13 +19,23 @@
     for src_path, docs_path in g.get_markdown_files(args.src_dir, exclude_dirs, doc_dir_path):
         # check to see if the files are the same !
         with open(src_path, "r") as src_f:
-            with open(docs_path, "r") as docs_f:
-                src_lines = src_f.readlines()
-                # fix image paths for docusaurus
-                for i in range(0, len(src_lines)):
-                    if "/docs/docusaurus/static/img" in src_lines[i]:
-                        src_lines[i] = src_lines[i].replace("/docs/docusaurus/static/img", "/img")
+            src_lines = src_f.readlines()
```

Comment:
Nit: Is this diff meant to be in this PR? looks like result of a bad merge/rebase?